css-backgrounds-4/Overview.bs

```diff
@@ -895,12 +903,9 @@ Initial: none
 Applies to: all elements
 Inherited: no
 Percentages: N/A
-Computed value: either the keyword ''box-shadow-offset/none'' or
-    a list, each item consisting of four absolute lengths
-    plus a computed color and optionally also a ''box-shadow-position/inset'' keyword
+Computed value: see individual properties
 Animatable: by computed value,
-    treating ''box-shadow-offset/none'' as a zero-item list
-    and appending blank shadows (''transparent 0 0 0 0'')
+    appending blank shadows (''transparent 0 0 0 0'')
```

Comment:
Just dropping "treating `none` as a zero-item list" would change the behavior.

I think that `box-shadow-offset` should treat `none` as `0 0` when interpolated with non-`none` values.

Reply:
The idea was to align it with `animation-name`, i.e. a value of `none` cancels out this one shadow within the list.

And I think it is better to treat `none` as-is, meaning the interpolation between `none` and non-`none` values is discrete and the shadow appears or disappears.

Sebastian

Reply:
Well this would break https://software.hixie.ch/utilities/js/live-dom-viewer/saved/11442 which works with interoperability, so you need CSSWG resolution to change it.